src/main/java/com/google/sps/data/ProfDataHolder.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProfDataHolder {
```

Comment:
Should probably note somewhere here (at least a comment) that this is per term. Maybe renaming the class to something like ProfDataForTerm, etc would be better though...